osidb/serializer.py

```diff
@@ -1283,9 +1283,11 @@ def update_trackers(self, old_affect, new_affect):
             promote = True  # remember to only regenarate Jira trackers
 
         for tracker in new_affect.trackers.all():
-            # no tracker updates for the closed ones
-            # because we consider these already done
-            if tracker.is_closed:
+            # Update security levels on closed tracker on unembargo
+            is_unembargoing = (
+                old_affect.flaw.is_embargoed and not new_affect.flaw.is_embargoed
```

Comment:
Will this actually work? :thinking: You have two instances of affect, e.g. the `old_affect` and `new_affect` but you are actually accessing its parent `flaw` only in this function, so for both `old_affect` and `new_affect` the `flaw` is queried just now and thus this will probably almost evaluate to False? :thinking:

Reply:
It seemed to work in the testing I did, but probably worth looking at more closely to make sure it is doing what I thought it was doing